### DISCUSSION TITLE
Is there a command for stopping all SO processes?

### DISCUSSION THREAD

**Discussion by hershmaster** [Security-Onion-Solutions/securityonion#1767] [Unsupported Versions] [answered]
Security Onion Version 2.3.2
OS: CentOS 7
Source: ISO
Architecture: Standalone

Is there a command or procedure for stopping all SO processes on 2.3.2?

_Background_
I installed the new SO on my old SO box today in my home lab, which has 2 hard drives in it (sda [1TB] and sdb[2TB]). During the initial installer phase I assigned sda for where the OS should go, and sdb where nsm should go.  After getting through all the install and setup, I logged in and found my sdb drive had not been mounted or formatted or anything (maybe a bug I should log, but don't have time to run through the install again right now to confirm).

My plan is to manually config sdb to hold the /nsm partition (had to do it before on the old setup due to my own typo). I want to make sure I stop all writing to the current /nsm so I can copy the files I've got there now to the new drive, then configure the mount and etc. But I'm not finding a way to cleanly stop everything.

Thanks in advance.

-Steve Hershman

**Comment by TOoSmOotH** [comment, marked as answer]
Try:
`sudo service salt-minion stop`  
`sudo service salt-master stop`  
`sudo service docker stop`  
`sudo mv nsm nsm.old`
do your stuff so that nsm mounts to your new disk
`sudo rsync -av /nsm.old/ /nsm/`
`sudo service docker start`
`sudo service salt-master start`
`sudo service salt-minion start`

When the minion service starts it will bring up all the components. I will test the install code again to make sure it is still using both disks but it was working fine as of a couple of days ago.

**Reply by hershmaster** [reply]
Thanks for those commands.  I've tried another couple of installs since my first, and I did see an option in the initial installer that I missed before that I think caused my install to skip over sdb (Installation Destination -> "Select all" option; I'd only chosen sda before).

I'm running into another issue with LVM making funky partition sizes, so I'm going to try again later and manually set up the drives. Definitely going to save those commands for later.
